recipes/vtkbool/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set version = "3.0rc1" %}
+{% set vtkbool_version = "3.0-rc1" %}
+
+package:
+  name: vtkbool
+  version: {{ version }}
+
+source:
+  url: https://github.com/zippy84/vtkbool/archive/refs/tags/v{{ vtkbool_version }}.tar.gz
+  sha256: a422662a7679b6f7e433596c4f80e1569e2b1c55b6a277c008881cd3ca4f7e1a
+  patches:
+    - modify-cmakelists.patch  # patch cmakelists to support building in conda env
+
+build:
+  number: 0
```

```suggestion
build:
  number: 0
  run_exports:
    - {{ pin_subpackage( vtkbool, max_pin='x.x.x' ) }}
```
Because you're shipping headers, you need a run_export which ensures that packages which build against this package get a compatible version at runtime. Set the max_pin according to ABI guarantees. Assume 'x.x.x' if there are no guarantees.